low-level/files/opt/seagate/sspl/setup/generate_resource_health_view/resource_health_view

```diff
@@ -139,149 +139,115 @@ class SSPLHealthView(object):
             self.RESOURCE_HEALTH_VIEW_PATH = args.path
             if os.path.exists(args.path):
                 self.RESOURCE_HEALTH_VIEW_PATH = args.path if args.path.endswith('/') else args.path+'/'
-                if args.support and args.encl:
+                if args.encl:
                     self.ENCL_MANIFEST_PATH = self.RESOURCE_HEALTH_VIEW_PATH + "encl_manifest.json"
```

Comment:
Why we are still referring to this code here? I think we have decided to move this code to the manifest script.

Reply:
removed the code